refs/biblio.json

```diff
@@ -596,17 +596,33 @@
         "aliasOf": "RFC6090"
     },
     "ECMA-262": {
-        "date": "June 2011",
-        "href": "http://www.ecma-international.org/publications/standards/Ecma-262.htm",
+        "title": "ECMA-262 6th Edition, The ECMAScript 2015 Language Specification",
```

Comment:
I would suggesting aliasing the whole of `[ECMA-262]` to `[ECMASCRIPT]` and adding "https://people.mozilla.org/~jorendorff/es6-draft.html" as an `"edDraft"`